prow/external-plugins/needs-rebase/main.go

```diff
@@ -121,7 +125,7 @@ func main() {
 	}, o.updatePeriod)
 
 	mux := http.NewServeMux()
-	mux.Handle("/", server)
+	mux.Handle("/", handler)
 	externalplugins.ServeExternalPluginHelp(mux, log, plugin.HelpProvider)
```

Comment:
All of this is entirely shared, too -- a user ideally could create an external plugin and provide only the bits that are different

Reply:
We can add the ability to pass only the `HelpProvider` func in the handler.